### PR TITLE
Make student solutions hidden behind a display toggle

### PR DESCRIPTION
**Pull request by morrowdavidson**
All of the student solutions on the project pages are now inside a `<details>` element.

### CODE REVIEW

File: javascript/frameworks/frameworks-project.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ The best way to actually learn something is to _use_ it! We've directed you to b
 
 Submit a solution with a pull request to this [file](https://github.com/TheOdinProject/curriculum/blob/master/javascript/frameworks/frameworks-project.md) on the Javascript Curriculum github repository. See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
+<details markdown="block">
+  <summary> Show Student Solutions </summary>
+
```
Comment (leila-alderman) [inline javascript/frameworks/frameworks-project.md]: Looks like you missed a `</details>` here.

File: rails_programming/databases_and_activerecord/project_ar_basics.md [hunk 2]
```diff
@@ -118,6 +118,9 @@ If any of those don't work, double check your associations.  Sometimes the error
 ### Student Solutions
 Send us your solution so we can show others! Submit a link to the Github repo with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).  Please include your partner's github handle somewhere in the description if they would like attribution.
 
+<details markdown="block">
+  <summary> Show Student Solutions </summary>
+
```
Comment (leila-alderman) [inline rails_programming/databases_and_activerecord/project_ar_basics.md]: Looks like you missed a `</details>` here.

File: ruby_programming/basic_ruby/project_advanced_building_blocks.md [hunk 3]
```diff
@@ -35,6 +35,9 @@ There's also [an entry on Bubble Sort on Wikipedia](http://en.wikipedia.org/wiki
 ### Student Solutions
 Submit a link below to this [file](https://github.com/TheOdinProject/curriculum/blob/master/ruby_programming/basic_ruby/project_advanced_building_blocks.md) on the ruby course github repo with your files in it by using a pull request. See the section on [Contributing](https://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
+<details markdown="block">
+  <summary> Show Student Solutions </summary>
+
 <details markdown="block">
   <summary> Show Student Solutions </summary>
```
Comment (leila-alderman) [inline ruby_programming/basic_ruby/project_advanced_building_blocks.md]: Looks like this file already had the toggle. Could you remove the duplication here?

**Comment by morrowdavidson** [comment]
Fixed with @leila-alderman corrections. Thanks @leila-alderman for looking it over.

File: ruby_programming/basic_ruby/project_advanced_building_blocks.md (hunk 3 above)
Comment (morrowdavidson) [reply ruby_programming/basic_ruby/project_advanced_building_blocks.md]: Not sure why the preview is showing it that way. But when I go to edit it only has one `<details>` element

Comment (leila-alderman) [reply ruby_programming/basic_ruby/project_advanced_building_blocks.md]: Weird. I'll trust the edit view over the diff view.

**Pull request merged**